pkg/application/application.go

```diff
@@ -24,14 +22,8 @@ const (
 )
 
 // GetDefaultAppName returns randomly generated application name with unique configurable prefix suffixed by a randomly generated string which can be used as a default name in case the user doesn't provide a name.
-func GetDefaultAppName(existingApps []preference.ApplicationInfo) (string, error) {
+func GetDefaultAppName() (string, error) {
```

Comment:
as we dont have the list of all the applications, can we get that from the cluster to make sure there is no conflict while generating a new name?

Reply:
there is no conflict now
Its just that the 2 components will belong to same application which is valid to be honest

Reply:
but then why did we have this check in the first place?

Reply:
I said "no conflict now" :)
But, previously app names were maintained in config file so, a create had to ensure app name generated is not something same as something already in config file :)
But its all removed from config file and there's nothing called `odo app create` so ensuring that the app name generated is unique is no longer valid
I hope this helps

Reply:
ack